store/src/app/products/components/product-card/product-card.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Product } from 'src/app/interfaces/products.interface';
+import { CartService } from '../../services/cart.service';
+
+
+@Component({
+  selector: 'app-product-card',
+  templateUrl: './product-card.component.html',
+  styleUrls: ['./product-card.component.scss']
+})
+export class ProductCardComponent implements OnInit {
+  constructor(private cartService: CartService) { };
```

Comment:
Incorrect order of parameters inside class. And also add access modifiers to your methods and properties

- Input
- Output
- public variarles
- protected variables
- private variables
- constructor
- lifecycle hooks methods
- static methods
- public methods
- private methods

Reply:
done